sdcm/remote/kubernetes_cmd_runner.py

```diff
@@ -42,6 +43,10 @@
 LOGGER = logging.getLogger(__name__)
 
 
+def is_scylla_bench_command(command):
```

Comment:
the logic should be backward
we should identify if it's cassnadra-stress of not.

anyhow we should write code that maps between a command and it's stress tool name, and configuration for docker image

we already have `DOCKER_IMAGE_PARAM_NAME`, we should at least use those.

I don't like that we are limiting this to support only scylla-bench

Reply:
At first, we have been supporting only `scylla-bench` without this PR. I haven't change the support matrix.
At second, I disagree that we `should` do anything more than really needed here, because it makes everything work as it was without `docker`.
I am ok to refactor stuff you propose only in a separate change when we are not blocked right now.

Reply:
Which use case it is blocking directly?

Reply:
https://github.com/scylladb/scylla-cluster-tests/issues/5646